coordinator/src/notifications.rs

```diff
@@ -115,20 +115,22 @@ fn build_notification(kind: &NotificationKind) -> fcm::Notification<'_> {
     let mut notification_builder = fcm::NotificationBuilder::new();
     match kind {
         NotificationKind::PositionSoonToExpire => {
-            notification_builder.title("Your position is about to expire");
-            notification_builder.body("Rollover your position for the next cycle.");
+            notification_builder.title("Your position is about to expire ⏳");
+            notification_builder
+                .body("Open your app to rollover your position for the next cycle.");
```

Comment:
The verb is "to roll over".

```suggestion
                .body("Open your app to roll over your position for the next cycle.");
```